lib/classes/CLI.js

```diff
@@ -8,9 +8,9 @@ const os = require('os');
 const chalk = require('chalk');
 
 class CLI {
-  constructor(serverless, isInteractive, loadedPlugins, inputArray) {
+  constructor(serverless, interactive, loadedPlugins, inputArray) {
     this.serverless = serverless;
-    this.isInteractive = isInteractive;
+    this.interactive = interactive;
```

Comment:
yep sometimes we want to explicitly set to interactive/non-interactive at the code level without changing the env vars (ie during tests) ... but actually the `isInteractive()` method should be aware of this probably, I'm gonna make that change.

And I had to rename it cause it was conflicting with the method name.